renaud.mk

```diff
@@ -1,4 +1,4 @@
 INSTANCE_ID = renaud
 VARS_FILE = vars_renaud.yaml
-
+PRINT_OUTPUT = /tmp
```

Comment:
And then I don't know for the others but I will set again the default value....
I the goal it to disable the print setting `PRINT_VERSION = NONE` Is a better way ...
see also: http://docs.camptocamp.net/c2cgeoportal/master/integrator/make.html#makefile-config-variables